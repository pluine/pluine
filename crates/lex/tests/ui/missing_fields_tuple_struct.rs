#[derive(pluine_lex_macros::Spanned)]
struct Foo();

fn main() {}
