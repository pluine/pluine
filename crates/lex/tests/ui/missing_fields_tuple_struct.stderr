error: no tuple fields supplied, at least one required
 --> tests/ui/missing_fields_tuple_struct.rs
  |
  | struct Foo();
  |           ^^
